### DISCUSSION TITLE
Wrong Layout on windows

### DISCUSSION THREAD

**Discussion by juneleung** [onweru/compose#41] [General]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

- [o] Read the README.md
- [o] Have the extended version of Hugo installed
- [o] Used the example site's config .toml as a reference

after guide with the readme doc, run the code
```
"git clone --recurse-submodules --depth 1 https://github.com/onweru/compose.git
cd compose/exampleSite/
hugo server --themesDir ../.."
```
then I open the http://127.0.0.1:1313/ page which shows the wrong layout, seems not to load the CSS corectlly. I try to make it on mac look work but failed on windows. How to solve this problem?

 - OS: [Windows 10]
 - Browser [chrome 88.0.4324.182]

**Comment by onweru** [comment]
@juneleung, did you succeed?

**Comment by juneleung** [comment]
@onweru  not yet, still can't load the CSS successfully. :(

**Comment by onweru** [comment]
1. Do you have [Hugo extended installed](https://gohugo.io/getting-started/installing/#chocolatey-windows) in your path?
2. If you have (1) above covered, have you [installed go](https://golang.org/doc/install)?

**Comment by juneleung** [comment]
Yeah, after I checked I had installed the Hugo-ext and go, I get the white bg of the web as attach, seems cant load the image successfully, How to solve the problem, Thanks!!!


![123](https://user-images.githubusercontent.com/32699460/111241945-2069e200-8639-11eb-8930-ed8c4e82bd86.png)

**Comment by onweru** [comment]
Ensure you read the docs. If you did, you must have seen:

1. https://docs.neuralvibes.com/docs/compose/install-theme/#option-3-great-for-testing-quickly
2. https://docs.neuralvibes.com/docs/compose/organize-content/

Beyond that, check the `config.toml` and ensure all you image paths are correct and that you have the logos in the right directory https://gohugo.io/getting-started/directory-structure/

I sense you're new to Hugo and this is not an issue with the theme, you just need to read the theme docs and hugos docs.
